draft-ietf-moq-transport.md

```diff
@@ -604,8 +606,13 @@ code, as defined below:
 * Duplicate Track Alias: The endpoint attempted to use a Track Alias
   that was already in use.
 
-* Too Many Subscribes: The session was closed because the subscriber used
-  a Subscribe ID equal or larger than the current Maximum Subscribe ID.
+* Too Many Requests: The session was closed because the subscriber used a
```

Comment:
```suggestion
* Too Many Requests: The session was closed because the endpoint used a
```

Reply:
@suhasHere I don't plan to update this branch anymore, but will make this change on moq-wg/moq-transport 905 instead.